src/components/ContentPreview/ReloadNotification.scss

```diff
@@ -0,0 +1,9 @@
+@import '../variables';
+
+.bcpr {
+    .bcpr-notification {
+        left: 50%;
```

Comment:
None of the typical horizontal centering stuff works e.g. text-align, margin:auto?